modules/dialog/dlg_replication.c

```diff
@@ -1048,24 +1040,25 @@ static void repl_prof_utimer_f(utime_t ticks, void *param)
 {
 #define REPL_PROF_TRYSEND() \
 	do { \
+		nr++; \
 		if (ret > repl_prof_buffer_th) { \
 			/* send the buffer */ \
-			dlg_replicate_profiles(); \
-			replicated = 1; \
+			if (nr) \
```

Comment:
Note that this has been changed in the latest version, to send a packet even if there are no profiles to be replicated. This has been changed to make sure that there is communication between nodes, event if there is no traffic. If this has been fixed on the clusterer module, using heartbeats, then ignore this comment.